13/umbraco-deploy/deploy-settings.md

```diff
@@ -98,7 +98,7 @@ However, we are aware that some customers prefer the option to use the backoffic
 This setting allows you to exclude a certain type of entity from being deployed. This is **not** recommended to set, but sometimes there may be issues with the way a custom media fileprovider works with your site and you will need to set it for media files. Here is an example:
 
 ```json
-"ExcludedEntityTypes": ['media-file'],
+"ExcludedEntityTypes": ["media-file"],
```

Comment:
should this be changed only for v13 or for all versions?

Reply:
This should be updated for all versions, because for JSON to be 100% valid, it needs to use double quotes 😄